usersession/agent/rest_api.go

```diff
@@ -179,20 +179,28 @@ type dummyReporter struct{}
 
 func (dummyReporter) Notify(string) {}
 
-func postServiceControl(c *Command, r *http.Request) Response {
+func validateJsonRequest(r *http.Request) (bool, Response) {
```

Comment:
Nitpick: this could be validateJSONRequest. Usually initialisms are written in uppercase, at least in Go's std library (e.g., `MarshalJSON` and `ServeHTTP`) 
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I'll wait for the CI to finish, then I'll change it :-)

Reply:
Thanks! :slightly_smiling_face:

Reply:
Done. :-)